song_match/song/songs/mary_had_a_little_lamb.py

```diff
@@ -10,26 +10,34 @@
 from song_match.song import Song
 from song_match.song.note import Note
 
+EIGHTH_NOTE = .25 # time for eighth note
+QUARTER_NOTE = .5 # time for quarter note
+HALF_NOTE = 1 # time half note
+WHOLE_NOTE = 2 # time for whole note
```

Comment:
1. I would move all these constants to above the `Note` class in `note.py`.

2. Also, if you add a colon `:` after the pound sign `#`, then Sphinx will generate documentation for these.

For example, `QUARTER_NOTE = .5  #: Time for quarter note`

Reply:
Thanks for the tip for Sphinx. When I remove these constant definitions from the MHALL class and put them in the note, I can't use them while passing them to the note constructors, i.e.             Note('E4', HALF_NOTE) without it throwing an error

Reply:
Did you import them?

After you move them to `note.py`, adding `from song_match.song.note import QUARTER_NOTE` to the top of the file should fix your error.

Reply:
Oh ok that worked. Thank you!